src/client/store/data/stateType.ts

```diff
@@ -1,13 +1,19 @@
 import { AssessmentName, CommentableDescriptionValue, CycleName, DataSourceLinked, SectionName } from '@meta/assessment'
-import { TableData } from '@meta/data'
+import { RecordAssessmentData } from '@meta/data'
 
 export interface DataBaseState {
-  tableData: TableData
   descriptions: Record<SectionName, Record<string, CommentableDescriptionValue>>
   linkedDataSources: Record<SectionName, Array<DataSourceLinked>>
 }
-export interface DataState {
-  [assessmentName: AssessmentName]: {
-    [cycleName: CycleName]: DataBaseState
+
+interface State1 {
+  tableData?: {
+    [assessmentName: AssessmentName]: {
+      [cycleName: CycleName]: RecordAssessmentData
+    }
   }
 }
+
+type State2 = Record<AssessmentName, Record<CycleName, DataBaseState>>
+
+export type DataState = State1 & State2
```

Comment:
what's State1 and State2? can't we find better names ? it's difficult to understand what those states type are